apps/routes/lib/route.ex

```diff
@@ -51,7 +51,7 @@ defmodule Routes.Route do
   @silver_line ~w(741 742 743 746 749 751)
   @silver_line_set MapSet.new(@silver_line)
 
-  @spec type_atom(t | type_int | String.t()) :: gtfs_route_type
+  @spec type_atom(t | type_int | String.t()) :: route_type
```

Comment:
`route_type` is defined a few lines above and is basically `gtfs_route_type` + `:the_ride`

```
@type route_type :: gtfs_route_type | :the_ride
```